Explorer++/Helper/Bookmark.h

```diff
@@ -124,7 +124,8 @@ class CBookmark
 {
 public:
 
-	CBookmark(const std::wstring &strName,const std::wstring &strLocation,const std::wstring &strDescription);
+	CBookmark(const std::wstring &strName,const std::wstring &strLocation,const std::wstring &strDescription); 
+	CBookmark(HKEY hKey, TCHAR *szKey);

```

Comment:
I think it would be better if named constructors were used. Something like:

`Create`
`Unserialize`
`UnserializeFromRegistry`